geth/client.go

```diff
@@ -491,6 +494,39 @@ func (c *Connector) SendTransaction(rawTx []byte) error {
 	return nil
 }
 
+// ConfirmedBalance returns number of funds available under control of
+// connector.
+//
+// NOTE: Part of the common.Connector interface.
+func (c *Connector) ConfirmedBalance(account string) (string, error) {
+	m := crypto.NewMetric(c.cfg.DaemonCfg.Name, string(c.cfg.Asset),
+		MethodConfirmedBalance, c.cfg.Metrics)
+	defer m.Finish()
+
+	balance := "0"
+
+	err := c.db.Update(func(tx *bolt.Tx) error {
```

Comment:
Why it is an Update, are we updating something in database?

Reply:
So this should be `View`? Fixed.